src/components/tests/App.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import App from '../App';
+
+test('Test App render successfully', () => {
```

Comment:
I think test description should leave the word **Test** out, like this:

```javascript
test('App render successfully', () => {});
```

Reply:
Good idea! I'll try it !